tests/base_tests.py

```diff
@@ -289,19 +294,39 @@ def get_access_requests(self, username, ds_type, ds_id):
     def logout(self):
         self.client.get("/logout/", follow_redirects=True)
 
+    def grant_access_to_dashboard(self, dashboard, role_name="Public"):
```

Comment:
Let's make `role_name` mandatory, using the default will cause confusion down the line.